src/applications/validate-mhv-account/components/errors/UpgradeAccountFailed.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MessageTemplate from '../MessageTemplate';
+
+const UpgradeAccountFailed = ({ upgradeAccount }) => {
```

Comment:
Should this be more consistent with `CreateMHVAccount` by making it a container that maps `upgradeMHVAccount` to its own prop?